src/db/utils/jobs/migration/ConvertTickDateClimbedFromStringToDate.ts

```diff
@@ -0,0 +1,18 @@
+import { connectDB, } from '../../../index.js'
+import { logger } from '../../../../logger.js'
+import { getTickModel } from '../../../TickSchema.js'
+
+/**
+ * Converts dateClimbed field on ticks from string to date.
+ */
+const onConnected = async (): Promise<void> => {
+  logger.info('Migrating ticks...')
+  const tickModel = getTickModel()
+
+  await (await tickModel.updateMany(
+    { dateClimbed: { $exists: true } },
+    { $set: { dateClimbed: { $toDate: '$dateClimbed' } } }
```

Comment:
Timezone here is a bit tricky. Naive version is to use UTC. But most ticks are probably for climbs in the US, so a first approximation may be to use some US timezone. And then the most labor intensive way to is to discern the timezone where the climb is and adjust accordingly.